app/styles/application.scss

```diff
@@ -0,0 +1,21 @@
+div.top {
+  display: flex;
```

Comment:
One thing that I forgot to tell ya is that all css rules should be alphabetized.